mid.go

```diff
@@ -26,8 +26,8 @@ func testEmptyInitialAllTx(rpcClient *LimeClient) testable {
 
 func testExampleTxFetching(rpcClient *LimeClient) testable {
 	return func() bool {
-		rlpString := "f90110b842307839623266366133633265316165643263636366393262613636366332326430353361643064386135646137616131666435343737646364363537376234353234b842307835613537653330353163623932653264343832353135623037653762336431383531373232613734363534363537626436346131346333396361336639636632b842307837316239653262343464343034393863303861363239383866616337373664306561633062356239363133633337663966366639613462383838613862303537b842307863356639366266316235346433333134343235643233373962643737643765643465363434663763366538343961373438333230323862333238643464373938"
-		res, err := rpcClient.GetEth(rlpString, "")
+		transactionHashes := "?transactionHashes=0x9b2f6a3c2e1aed2cccf92ba666c22d053ad0d8a5da7aa1fd5477dcd6577b4524&transactionHashes=0x5a57e3051cb92e2d482515b07e7b3d1851722a74654657bd64a14c39ca3f9cf2&transactionHashes=0x71b9e2b44d40498c08a62988fac776d0eac0b5b9613c37f9f6f9a4b888a8b057&transactionHashes=0xc5f96bf1b54d3314425d2379bd77d7ed4e644f7c6e849a74832028b328d4d798"
```

Comment:
Maybe use it singular `transactionHash` instead of `transactionHashes`

Reply:
I wrote it in plural because I'm sending a list of transaction hashes instead of a single one. This leaves `transactionHash` open as a variable name for when you are looping through the list.